Documentation/ApiOverview/TSFE/Index.rst

```diff
@@ -0,0 +1,216 @@
+.. include:: /Includes.rst.txt
+.. index:: TSFE; TypoScriptFrontendController
+.. _tsfe:
+
+====
+TSFE
+====
+
+What is TSFE?
+=============
+
+TSFE is short for :php:`\TYPO3\CMS\Frontend\Controller\TypoScriptFrontendController`,
+a class which exists in the system extension EXT:frontend.
+
+As the name implies: A responsibility of TSFE
+is page rendering. It also handles reading from and writing to the page cache.
+For more details it is best to look in the source code.
+
+There are several contexts in which the term TSFE is used:
+
+1.  PHP: It is passed as request attribute
```

Comment:
I suggest an unordered list, as the points have no order.

Reply:
done